### PR TITLE
office-js: Outlook API in preview

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/office-js/index.d.ts [hunk 1]
```diff
@@ -8271,7 +8287,7 @@ declare namespace Office {
          *
          * When including links in HTML markup, you can disable online link preview by setting the id attribute on the anchor (<a>) to "LPNoLP" (please see the Examples section for a sample).
          *
-         * [Api set: Mailbox 1.1]
+         * [Api set: Mailbox 1.2]
```
Comment (AlexJerabek) [inline types/office-js/index.d.ts]: Can you make sure to change this in the legacy docs afterwards? I can show you how to do that without rerunning everything.

File: types/office-js/index.d.ts [hunk 2]
```diff
@@ -8691,7 +8707,41 @@ declare namespace Office {
...
+     * @remarks
+     * <table><tr><td>{@link https://docs.microsoft.com/outlook/add-ins/understanding-outlook-add-in-permissions | Minimum permission level}</td><td>ReadItem</td></tr>
+     * 
+     * <tr><td>{@link https://docs.microsoft.com/outlook/add-ins/#extension-points | Applicable Outlook mode}</td><td>Compose</td></tr></table>
+     * 
+     * @beta
+     */
+    export interface Organizer {
+        /**
+         * Gets the organizer value of an appointment.
```
Comment (AlexJerabek) [inline types/office-js/index.d.ts]: Consider being explicit about the returned type in this first line.

File: types/office-js/index.d.ts [hunk 3]
```diff
@@ -8691,7 +8707,41 @@ declare namespace Office {
     }
 
     /**
-     * The subclass of {@link Office.Item} dealing with apppointments.
+     * Provides a method to get the organizer value of an appointment in an Outlook add-in.
```
Comment (AlexJerabek) [inline types/office-js/index.d.ts]: Is there also a description about the class itself that would be appropriate here? Maybe defining an Organizer?

File: types/office-js/index.d.ts [hunk 4]
```diff
@@ -8691,7 +8707,41 @@ declare namespace Office {
...
+     * 
+     * @beta
+     */
+    export interface Organizer {
+        /**
+         * Gets the organizer value of an appointment.
+         * 
+         * The getAsync method starts an asynchronous call to the Exchange server to get the organizer value of an appointment.
+         * 
+         * The organizer value of the item is provided as an {@link Office.EmailAddressDetails} in the asyncResult.value property.
```
Comment (AlexJerabek) [inline types/office-js/index.d.ts]: Consider merging these sentences into one paragraph.

File: types/office-js/index.d.ts [hunk 5]
```diff
@@ -8807,6 +8857,22 @@ declare namespace Office {
          * <tr><td>{@link https://docs.microsoft.com/outlook/add-ins/#extension-points | Applicable Outlook mode}</td><td>Appointment Organizer</td></tr></table>
          */
         optionalAttendees: Recipients;
+        /**
+         * Gets the email address of the organizer for the specified meeting.
+         *
+         * The organizer property returns an Organizer object that provides a method to get the organizer value.
```
Comment (AlexJerabek) [inline types/office-js/index.d.ts]: Consider merging these sentences to provide a clearer picture of what this property is actually doing.

**Review by AlexJerabek — APPROVED** [review]
There's some structural considerations, but the content looks good.

File: types/office-js/index.d.ts (hunk 1 above)
Comment (ElizabethSamuel-MSFT) [reply types/office-js/index.d.ts]: For sure.

**Review by AlexJerabek — APPROVED** [review]

**Pull request merged**